roles/dtc/common/templates/ndfc_attach_vrfs_loopbacks.j2

```diff
@@ -0,0 +1,18 @@
+[
+  {% for attach in MD_Extended.vxlan.overlay_services.vrf_attach_groups_dict[vrf["attach_group"]] %}
+  {
+    "fabric": "{{ MD.vxlan.global.name }}",
+    "vrfName": "{{ vrf["name"] }}",
+    {% for switch in MD.vxlan.topology.switches %}
+      {% if switch.management.management_ipv4_address == attach["hostname"] %}
+    "serialNumber": "{{ switch.serial_number }}",
+      {% endif %}
+    {% endfor %}
+    "freeformConfig": "",
+    "extensionValues": "",
+    "vlan": "{{ vrf["vlan_id"] }}",
+    "deployment": true,
```

Comment:
@mthurstocisco this was the deployment question

Reply:
updated, think this just got copied across from our old solution